addons/mail/static/src/core/common/picker_content.scss

```diff
@@ -7,3 +7,18 @@
         height: 100% !important;
     }
 }
+
+.popover-small-ui-picker {
+    bottom: 0!important;
+    top:auto !important;
+    left: 0!important;
+
+    .o-mail-PickerContent{
+        width: 100%;
+
+        .o-Emoji {
+            width: 10%;
```

Comment:
By using this you are restricting the number of emojis could be visible in a single row. Becuase of this max 10 emojis will be visible in a row no matter the size of the window.